### ISSUE TITLE
Robotloader version collision

### ISSUE THREAD

**Issue by Lexinek** [rectorphp/rector#341]
Hi, i can not use rector/rector on my project due to nette/robot-loader versions conflict :(
```
Problem 1
    - Installation request for rector/rector dev-master -> satisfiable by rector/rector[dev-master].
    - rector/rector dev-master requires nette/robot-loader ^3.0 -> satisfiable by nette/robot-loader[3.0.x-dev, v3.0.0, v3.0.1, v3.0.2, v3.0.3] but these conflict with your requirements or minimum-stability.
```

Just curious, how does rector use robot loader? It can not be replaced with PSR4 autoloading right?

**Comment by Lexinek** [comment]
Anyways i will probably be able to refactor my project and drop robot-loader dependency so it will be possible to use 3.0, but still curious how is robot-loader used here

**Comment by TomasVotruba** [comment]
Hey, there is no need to resolve local deps to match Rector. That would be too strict requirement for user :)
Ideal way to solve this is #319 , that is WIP.

For now it's recommended this solution: https://github.com/rectorphp/rector#do-you-have-conflicts

How does it work?

**Comment by Lexinek** [comment]
Yeah, now i have conflict with phpstan with the `nikic/php-parser`, i will try the method described in config.

Im reviving project that was written in nette 2.3 and i found some ugly code, i will need a lot of rectoring 😄 it is crazy how 1 year personal progress can change code insights

**Comment by TomasVotruba** [comment]
Nice :palm_tree: 

Drop me a share on Github, I could review what can and can't be done with Rector

**Comment by Lexinek** [comment]
I will add ideas to rector issue.

This one was solved by installing phpstan/phpstan-shim instead of phpstan/phpstan. 
Looking forward to have rector-shim 😉
